performance/benchmarks/CMakeLists.txt

```diff
@@ -14,9 +14,6 @@ foreach(benchmark ${benchmarks})
   target_link_libraries(${name} Threads::Threads)
   target_link_libraries(${name} benchmark)
   list(APPEND ALL_BENCHMARKS ${name})
-
-  # surpresses the "'CSVReporter' is deprecated" errors when compiling benchmark.cc
-  set_target_properties(${name} PROPERTIES COMPILE_OPTIONS "-Wno-deprecated-declarations" )
 endforeach()
 
 # one target to execute all benchmarks with the user flags
```

Comment:
If you are already changing the file already, could you put the add_custom_target into an if-then-else block. Because the python benchmarks should be only executed if the bindings are actually build. There is the case that someone has installed the python library but does not build them, where this if-then-else still ignores the python benchmarks, but I think it is more important to cover the other case, which seems to me more like average user behaviour
```
if(BUILD_BINDINGS)
  add_custom_target(
      benchmarks
      COMMAND ${CMAKE_CURRENT_BINARY_DIR}/benchmark_interpolator
              --benchmark_counters_tabular=true
              ${RUN_BENCHMARKS_FLAGS}
      # Assume bash-like shell to pass PYTHONPATH
      COMMAND "PYTHONPATH=${CMAKE_BINARY_DIR}" ${PYTHON_EXECUTABLE}
              ${CMAKE_CURRENT_SOURCE_DIR}/benchmark_spherical_representations.py
      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
      DEPENDS ${ALL_BENCHMARKS}
  )
else()
  add_custom_target(
      benchmarks
      COMMAND ${CMAKE_CURRENT_BINARY_DIR}/benchmark_interpolator
              --benchmark_counters_tabular=true
              ${RUN_BENCHMARKS_FLAGS}
      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
      DEPENDS ${ALL_BENCHMARKS}
  )
endif()
```

Reply:
Yes, good point !

Reply:
Done!